core/plugins/stack/python/version.ts

```diff
@@ -40,6 +40,23 @@ file with a version inside, like "3.9".
 See https://github.com/pyenv/pyenv
 `;
 
+/*Search for a setup.py file with python version*/
```

Comment:
Can you add a description on the same format as the `pipfile` and `pyenv` function?

![image](https://user-images.githubusercontent.com/30262244/146936008-aca0f5dc-f599-4a12-aadd-addd474506ed.png)

Also make sure the function shows its definition when hover over it:

Doesn't show:
![image](https://user-images.githubusercontent.com/30262244/146936162-c79b2714-2103-4941-a609-6ea8e959dc5d.png)

How should be:
![image](https://user-images.githubusercontent.com/30262244/146936333-0b1568ec-150a-49fc-8aab-d35860e019db.png)